docs/src/main/paradox/cdi.md

```diff
@@ -0,0 +1,72 @@
+# Context Injection
+
+As we saw in the Getting Started example, it is perfectly possible to inject any Bean available in Spring into a 
+Cloudstate entity class.
+It is also possible to use the annotations present in the javax.inject package 
+([JSR330](https://jcp.org/en/jsr/detail?id=330)). 
+The Cloudstate Springboot Support library already includes the necessary dependencies so you don't have to worry about it.
+
+You can annotate your entity classes with Spring @Component or @Service annotations but we have created a convenient 
+annotation that we call @CloudstateEntityBean that can be used for that too.
+
+## JSR330
+
+The Cloudastate Springboot Support library supports JSR330 within the scope of the support provided by Spring itself to 
```

Comment:
`Cloudastate` -> `Cloudstate` and suggest consistently use `Spring Boot` everywhere in the text (not `Springboot`):

```suggestion
The Cloudstate Spring Boot Support library supports JSR330 within the scope of the support provided by Spring itself to 
```